From 2b5eefbedead5fd55cbfba25f57ea485edc99024 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Fri, 16 Sep 2022 14:59:19 +0800
Subject: [PATCH] Revert "Linux 5.10.143"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit f1101295c145e9710b1b37e9b0a13ef9af9af0c9.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 60b2018c26db..655fe095459b 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 143
+SUBLEVEL = 142
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.34.1

